### DISCUSSION TITLE
Problem using resteasy-multipart-provider  >6.0.0 with jakarta

### DISCUSSION THREAD

**Discussion by gzayat** [resteasy/resteasy#3676] [General]
I'm migrating a javax application to jakarta and updating to the latest version but every time I add the dependency above version 6.0.0 onwards the REST service stops working error 405, this looks like a dependency but I can't identify the reason so I forced to use version 5.0.7 and make a kludge to work.

**Comment by jamezp** [comment]
You'll need to fully convert your name space from `javax` to `jakarta` before version 6.x would work. Note I'd suggest 6.2.4.Final which is the latest.

Do your REST endpoints still use `javax.ws.rs.*`?

**Reply by gzayat** [reply]
Yes, I am using all versions dealing with jakarta, most imports have changed from javax. for jakarta.

I don't know why when I use version <= 5.0.7 the rest works even though jakarta is annotated in @path and the rest.

Now when I use version above 6 I start getting the 405 error for the same method that worked in 5.0.7

**Reply by jamezp** [reply]
What container are you using or do you use RESTEasy as a Java SE application?
